FieldOpt/Runner/main.cpp

```diff
@@ -1,7 +1,5 @@
 /******************************************************************************
  *
- *
```

Comment:
Since you have already changed this file: Maybe change the license header to be in the same format as the more recently uodated files (such as in `grid.h`, before you changed the line breaks).